config/src/config/state_sync_config.rs

```diff
@@ -121,10 +121,10 @@ impl Default for StorageServiceConfig {
             max_account_states_chunk_sizes: 1000,
             max_concurrent_requests: 4000,
             max_epoch_chunk_size: 100,
-            max_network_channel_size: 1000,
+            max_network_channel_size: 4000,
             max_transaction_chunk_size: 1000,
             max_transaction_output_chunk_size: 1000,
-            storage_summary_refresh_interval_ms: 100,
+            storage_summary_refresh_interval_ms: 50,
```

Comment:
what's the trade-off for refreshing more frequently? Is it just require more CPU?

Reply:
This is the refresh rate of the local storage. It'll just require more disk reads. But, these reads are very cheap, so doing them at 50ms intervals shouldn't bad 😄